examples/tasks/mock-statistics-file.json

```diff
@@ -0,0 +1,40 @@
+{
+	"version": 1,
+	"schedule": {
+		"type": "simple",
+		"interval": "1s"
+	},
+	"max-failures": 10,
+	"workflow": {
+		"collect": {
+			"metrics": {
+				"/intel/mock/foo": {},
```

Comment:
These files are an alternative option if we don't want to use psutil. It probably makes sense to delete them since the .sh files don't ever call them.